pkg/core/native/gas_record.go

```diff
@@ -0,0 +1,13 @@
+package native
+
+import "math/big"
+
+// GASIndexPair contains block index together with generated gas per block.
+// It is used to cache NEO GASRecords.
+type GASIndexPair struct {
```

Comment:
Does it need to be exported?

Reply:
No, we don't use it anywhere else. Fixed.